app/views/surveys/webform.pug

```diff
@@ -20,14 +20,26 @@ block script
     script#main-script(defer, module, src=`${basePath}${offlinePath || ''}/js/build/enketo-webform${suffix}-bundle.min.js`)
   else
     script#main-script(defer, module, src=`${basePath}${offlinePath || ''}/js/build/enketo-webform${suffix}-bundle.js`)
-  script(nomodule).
-    var mainScript = document.querySelector( '#main-script' );
-    var mainScriptSrc = mainScript.src;
-    mainScript.remove();
-    var script = document.createElement( 'script' );
-    script.type = 'text/javascript';
-    script.src = mainScriptSrc.replace('-bundle.', '-ie11-bundle.');
-    document.body.appendChild( script );
+  script.
+    if ((/MSIE 10/i.test(navigator.userAgent)) || (/MSIE 9/i.test(navigator.userAgent) || /rv:11.0/i.test(navigator.userAgent)) || (/Edge\/\d./i.test(navigator.userAgent))) {
```

Comment:
Please stick to IE11 and Edge only. A hard fail would be better for IE9 and IE10 since Enketo definitely won't work correctly in those browsers.

Reply:
got it, code updated